lab-matt/entry.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+require('!!file?name=[name].[ext]!./html/index.html');
+require('./scss/base.scss');
+
+const angular = require('angular');
+const cowsay = require('cowsay-browser');
+const app = angular.module('cowsayApp', []);
+
+
+app.controller('CowsayController', ['$scope', function($scope){
+  $scope.cowsayText = 'with I had something better to say...';
+  $scope.cowsayFile = 'default';
```

Comment:
For your say() function, the better practice for a live preview would be to just return the cowsay.say output. Doing this will require some adjustments to you html.